actions/earned-income-budgeting.vbs

```diff
@@ -360,697 +883,729 @@ End If
+					If type_of_job = "T" Then EARNED_INCOME_PANELS_ARRAY(income_type, the_panel) = "T - Training Program"
+					If type_of_job = "P" Then EARNED_INCOME_PANELS_ARRAY(income_type, the_panel) = "P - Service Program"
+					If type_of_job = "R" Then EARNED_INCOME_PANELS_ARRAY(income_type, the_panel) = "R - Rehab Program"
+
+					'formatting the information from the panel and adding it to the EARNED_INCOME_PANELS_ARRAY
+					EARNED_INCOME_PANELS_ARRAY(income_verif, the_panel) = trim(job_verif)
+					EARNED_INCOME_PANELS_ARRAY(employer, the_panel) = replace(employer_name, "_", "")
+					EARNED_INCOME_PANELS_ARRAY(employer_with_underscores, the_panel) = employer_name
+					EARNED_INCOME_PANELS_ARRAY(hourly_wage, the_panel) = trim(listed_hrly_wage)
+					EARNED_INCOME_PANELS_ARRAY(income_start_dt, the_panel) = replace(start_date, " ", "/")
+					EARNED_INCOME_PANELS_ARRAY(income_end_dt, the_panel) = replace(end_date, " ", "/")
+					If EARNED_INCOME_PANELS_ARRAY(income_start_dt, the_panel) = "__/__/__" Then EARNED_INCOME_PANELS_ARRAY(income_start_dt, the_panel) = ""
+					If EARNED_INCOME_PANELS_ARRAY(income_end_dt, the_panel) = "__/__/__" Then EARNED_INCOME_PANELS_ARRAY(income_end_dt, the_panel) = ""
+					EARNED_INCOME_PANELS_ARRAY(old_verif, the_panel) = trim(current_verif)
+					' If frequency = "1" Then EARNED_INCOME_PANELS_ARRAY(pay_freq, the_panel) = "1 - One Time Per Month"      'setting full detail to the array instead of a single letter code
+					' If frequency = "2" Then EARNED_INCOME_PANELS_ARRAY(pay_freq, the_panel) = "2 - Two Times Per Month"
+					' If frequency = "3" Then EARNED_INCOME_PANELS_ARRAY(pay_freq, the_panel) = "3 - Every Other Week"
+					' If frequency = "4" Then EARNED_INCOME_PANELS_ARRAY(pay_freq, the_panel) = "4 - Every Week"
+					' If frequency = "5" Then EARNED_INCOME_PANELS_ARRAY(pay_freq, the_panel) = "5 - Other"
+
+					EARNED_INCOME_PANELS_ARRAY(income_list_indct, the_panel) = "NONE"       'This is where all of the array items from LIST_OF_INCOME_ARRAY will be added that are associated with this panel
+					EARNED_INCOME_PANELS_ARRAY(this_is_a_new_panel, the_panel) = FALSE      'identifies if a panel was created by the script or not - these are currently existing - changes CNote
+
+					the_panel = the_panel + 1       'incrementing our counter to be ready for the next panel/member/income type
+				End If      'If save_this_panel = TRUE Then
+			Next            'For panel = 1 to number_of_jobs_panels
+		End If              'If number_of_jobs_panels <> "0" Then
+	Next                    'For each member in HH_member_array
+
+	'FUTURE FUNCTIONALITY - this will remove panels if ended long enough ago.
+	'TESTING NEEDED
+	If panels_to_delete <> "" Then                              'at this time panels_to_delete wil ALWAYS = "" so no need to comment out
+		array_of_ended_panels = split(panels_to_delete, "~")    'NEED COMMENTS
```

Comment:
1064: 'NEED COMMENTS" found. Are commented needed for this section of code?

Reply:
This is dead code, it is never called and is a future functionality. I removed the need comments because it needs lots of other things too.